README.md

```diff
@@ -20,71 +20,250 @@ Use W&B to organize and analyze machine learning experiments. It's framework-agn
 
 [Documentation →](https://docs.wandb.com)
 
```

Comment:
NOTE:
we now have two readme's.   a readme for the repo which is what you changed, and one for the package that is displayed on things like this: (package_readme.md)
https://pypi.org/project/wandb/

The package readme goes out in a release, so if you are going to change that, let me know and I would like to do the merge when it is safe.

Reply:
Gotcha. I wasn't planning on changing that, but I can do it if you think it would be helpful Jeff. Let me know.